tendrl/ceph_integration/manager/cluster_monitor.py

```diff
@@ -366,6 +375,27 @@ def on_sync_object(self, data):
                 " received for %s" % data['type']
             )
 
+    def _get_utilization_data(self):
+        from ceph_argparse import json_command
+        import rados
+        cluster_handle = rados.Rados(
+            name=ceph.RADOS_NAME,
+            clustername=self.name,
+            conffile=''
+        )
+        cluster_handle.connect()
+        prefix = 'df'
+        ret, outbuf, outs = json_command(
+            cluster_handle,
+            prefix=prefix,
+            argdict={'format': 'json'},
```

Comment:
format json doesn't provide total pool capacity and even % used We might need to use the plain normal command ceph df and parse the command output as a string.